docusaurus.config.js

```diff
@@ -126,6 +126,11 @@ const config = {
             to: "/blog/the-opentofu-fork-is-now-available",
             from: "/fork",
           },
+          // TODO: This will be possible after upgrading to Docusaurus 3
```

Comment:
Weird they don't have it already

Reply:
Yeah, that was strange, can't wait for v3 to be released